pnpm-lock.yaml

```
@@ -9818,7 +9824,7 @@ packages:
       '@babel/code-frame': 7.18.6
       '@babel/parser': 7.21.2
       '@babel/traverse': 7.21.2
-      '@babel/types': 7.20.7
+      '@babel/types': 7.21.2
```

```suggestion
      '@babel/types': 7.20.7
```
I don't know why this has changed, but it shouldn't have.

---

true, but nothing broke with it there, can't we keep it, higher number looks better 🥺

---

We _can_, but I'd rather not. It's not related to this PR and would make tracing any bugs harder than it needs to be.

---

I can't update it through the comment, because github doesn't accept the command.

I will move it to second PR, and change this one